src/ts/playerutils.ts

```diff
@@ -59,6 +59,29 @@ export namespace PlayerUtils {
     return player.getSeekableRange() && player.getSeekableRange().start || defaultValue;
   }
 
+  /**
+   * Returns seekable time range.
+   * For live streams it will calculate timestamp timerange if tweak
+   * `enable_seek_for_vod` is not present
+   */
+  export function getSeekableRange(player: PlayerAPI): TimeRange {
+    let {start, end} = player.getSeekableRange();
```

Comment:
The mobile SDKs will not return anything here.

Please see `getSeekableRangeStart` and respect accordingly.

Reply:
Hmm, I can do an additional check for covering `mobile SDK`, but not sure about using `getSeekableRangeStart`, it returns 0 by default? In what cases do we not get `start` and `end` here?

Reply:
Is there any documentation I can find regarding this on mobile SDK?

Reply:
No there is no documentation. 

But mobile SDK versions don't even implement `getSeekableRange`. They will return `undefined` which will return in a crash here.

Reply:
I see, will go with simple `!isLive()` check to return `seekableRange`, otherwise, it will get calculated. This calculation is only possible for live streams and is used for markers on live stream anyway.

Reply:
Updated https://github.com/bitmovin/bitmovin-player-ui/pull/310/commits/2e8de6c57c7fd4553f633acbe01eea708f9bbe4d